style.css

```diff
@@ -0,0 +1,298 @@
+@font-face {
+  font-family: 'Olde English';
+  src: url('Olde English.ttf');
+}
+
+header 
+{
+	position  : fixed;
+	padding   : 15px;
+	top       : 0px;
+	left      : 0px;
+	right     : 0px;
+	display   : flex;
+	height: 25px;
+	border-bottom: 1px solid #DDD;
+	background-color: white;
+}
+
+header *:hover {
+	cursor: pointer;
+}
+
+.NewYorkTitle
+{
+	font-family: 'Olde English';
+	font-size: 29px;
+	margin   :0;
+	margin-top: -5px;
+	margin-left: 20%;
+}
+header ul 
+{
+	margin: 0px;
+}
+header ul li 
+{
+   display: inline-block;
+   margin-right: 15px;
+
+}
+#logButton
+{
+	position : absolute;
+	top : 10px;
+	right: 20px;
+	margin-left : 12px;
+	margin-top : 10;
+}
+
+ button 
+{
+	padding : 10px;
+	background: #2980b9;
+	color : white;
+	font-weight: bold;
+	font-size: 10px;
+	border-radius: 3px;
+
+
+}
+#titleOfArticle {
+	margin-top: 100px;
+	display : block;
+	width : 53%;
+	margin-left: 100px;
+	font-family: georgia;
+}
+
+#container {
+	display: block;
+	margin: 100px auto;
+	width: 550px;
+}
+
+#image1 {
+	width: 800px;
+	display: block;
+	margin: 30px auto;
+}
+
+#image1 img {
+	height: 500px;
+	width: auto;
+}
+#biggerP {
+	font-size: 16px;
+	/*margin: 10px ;*/
+	width: 95%;
+	color: grey;
+}
+#smallerP {
+	font-size: 13px;
+	color: grey;
+	margin-top: -10px;
+}
+
+#smallGrey {
+	font-size: 15px;
+	color: grey;
+}
+
+	 h5 {
+	font-size: 17px;
+	color: grey;
+	font-weight: normal;
+	margin: 0px;
+}
+
+#video p {
+	font-size: 15px;
+	color: grey;
+	margin: 0px;
+}
+
+#img7 img{
+	width: 570px;
+	height: auto;
+}
+
+#img7 p {
+	font-size: 15px;
+	color: grey;
+	margin: 0px;
+}
+
+#ending {
+	font-size: 13px;
+	color: #ADB7BA;
+	margin: 20px 0;
+}
+
+p {
+	font-family: 'Times New Roman';
+	font-size: 20px;
+	color: #393737;
+}
+
+#date {
+	font-size: 14px;
+	color: #333;
+}
+
+
+.box {
+	display: inline-block;
+	margin: 7px;
+	width: 250px;
+	height: auto;
+}
+
+#sections {
+	display: inline;
+	float: right;
+	margin-right: 50px;
+	font-size: 20px;
+	width: 25%;
+	border-top: 1px solid black;
+}
+
+#sections li {
+	padding: 15px 0;
+	list-style: none;
+	border-top: 1px solid #D4D8D9;
+}
+
+.suggestSection {
+	float: left;
+	margin-left: 50px;
+	width: 63%;
+	margin-right: 0;
+	border-top: 1px solid black;
+
+}
+
+.suggestSection p {
+	display: inline-block;
+}
+
+.box {
+	width : 250px;
+	height: 300px;
+	vertical-align: top;
+	margin-bottom: 30px;
+}
+.box p
+{
+   margin : 10px;
+   height: 100px;
+   vertical-align: top;
+
+}
+h3	{
+	font-size: 17px;
+	margin-left: 10px;
```

Comment:
Why did you add this margin? This makes the headers move to the right and the consequence is they don't align left with the rest of the paragraph